PantryServer/Models/OrderDetail.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PantryServer.Models
+{
+    public class OrderDetail
+    {
+        public int OrderDetailId { get; set; }
+        public int OrderId { get; set; }
+        public int ProductId { get; set; }
+        public int Quantity { get; set; }
+        public decimal UnitPrice { get; set; }
+        public virtual Product product { get; set; }
+        public virtual Shop shop { get; set; }
```

Comment:
not sure if we need to reference shop - we have access to it from the product object.